src/ncp-spinel/SpinelNCPTaskSendCommand.cpp

```diff
@@ -403,6 +403,8 @@ nl::wpantund::SpinelNCPTaskSendCommand::vprocess_event(int event, va_list args)
 		check_noerr(mNextCommandRet);
 	}
 
+	syslog(LOG_ERR, "SendCommand task finish: mRetVal: %u mReturnValue");
```

Comment:
I think the `mReturnValue` is intended as parameter.

In general, do we need this? I think since it is at LOG_ERR level and it will be printed on every command, it will add a lot to the wpantund log. Suggest removing it.